src/main/java/teammates/common/datatransfer/StudentAttributes.java

```diff
@@ -130,7 +129,7 @@ public boolean isRegistered() {
     }
 
     public String getRegistrationUrl() {
-        return new Url(Config.APP_URL + Const.ActionURIs.STUDENT_COURSE_JOIN_NEW)
+        return new Url(Const.ActionURIs.STUDENT_COURSE_JOIN_NEW)
```

Comment:
I suggest either rename the method to add 'relative' and/or add a comment to explain its behavior.